app/index.js

```diff
@@ -41,14 +41,14 @@ module.exports = function App(config) {
   // Provide user feedback when verbose output is enabled
   app.log('info', 'verbose output enabled', true);
 
-  // Attach RouteManager to app object, the primary set of mockyeah API methods.
-  app.routeManager = new RouteManager(app);
-
-  app.use(bodyParser.json());
-
   app.get('/', (req, res) => {
     res.send('Hello, mockyeah!');
   });
 
+  app.use(bodyParser.json());
+
+  // Attach RouteManager to app object, the primary set of mockyeah API methods.
+  app.routeManager = new RouteManager(app);
+
```

Comment:
Had to move this down below the `/` route so our generic `*` route in the route manager didn't take precedence.

Reply:
As a side note - do we still want/need this fake index route?

Reply:
It is helpful at times to hit the root url to receive feedback that mockyeah is running. I find this especially useful during tutorials/demos.

Do you ever foresee a scenario that a route would need to be mounted on the root?

Reply:
@ryanricard I doubt there would be a scenario where mockyeah would need to mounted at the root. If you wanted to support that anyway, we could consider a more obscure health endpoint for mockyeah, e.g., `/__mockyeah__`.